web-ui/components/Editor.vue

```diff
@@ -61,6 +68,34 @@ export default {
     handleTab(tab) {
       this.tab = tab;
     },
+    startDrag() {
+      this.isResizing = true;
+      window.addEventListener("mousemove", e => this.resizeEditor(e));
+      window.addEventListener("mouseup", this.stopDrag);
+    },
+    stopDrag() {
+      window.removeEventListener("mousemove", e => this.resizeEditor(e));
+      window.removeEventListener("mouseup", this.stopDrag);
+      this.isResizing = false;
+    },
+    resizeEditor(e) {
+      let width = window.innerWidth - e.clientX;
+      let windowWidth =
+        window.innerWidth ||
+        document.documentElement.clientWidth ||
+        document.body.clientWidth;
```

Comment:
You should reverse these and reuse the windowWidth in both places as appose to getting the value two different ways.